its/src/test/java/com/sonar/it/csharp/IssuesOnMissingFilesTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SonarSource :: C# :: ITs :: Plugin
+ * Copyright (C) 2011-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonar.it.csharp;
+
+import com.sonar.it.shared.TestUtils;
+import com.sonar.orchestrator.build.BuildResult;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.sonarqube.ws.Issues;
+
+import java.util.List;
+
+import static com.sonar.it.csharp.Tests.ORCHESTRATOR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IssuesOnMissingFilesTest {
+  @ClassRule
+  public static final TemporaryFolder temp = TestUtils.createTempFolder();
+
+  private static final String PROJECT = "ProjectWithIssuesOnMissingFiles";
+  private static BuildResult buildResult;
+
+  @BeforeClass
+  public static void init() throws Exception {
+    TestUtils.reset(ORCHESTRATOR);
+    buildResult = Tests.analyzeProject(temp, PROJECT, null);

```

Comment:
could you also add `/warnaserror:AD0001` in `runMsBuild` in https://github.com/SonarSource/sonar-dotnet/blob/3f0918e3a32c8dc67079b7c5bd7f61ddb858df8d/its/src/test/java/com/sonar/it/shared/TestUtils.java#L148 ?

It would make sure that we never get analysis warnings in our Java ITs - and like this, this test would ensure that there are no uncaught errors for this scenario